### PR TITLE
Fix Form Linting

### PR DESCRIPTION
**Pull request by philippfromme**
* add latest Camunda versions to form linter
* sort lint errors alphabetically

NciZFR29k1

---

Closes https://github.com/camunda/camunda-modeler/issues/2880
Closes https://github.com/camunda/camunda-modeler/issues/2885

### CODE REVIEW

**Comment by philippfromme** [comment]
# Artifacts

* https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/linting-final-final/camunda-modeler-linting-final-final-linux-x64.tar.gz
* https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/linting-final-final/camunda-modeler-linting-final-final-mac.dmg
* https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/linting-final-final/camunda-modeler-linting-final-final-mac.zip
* https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/linting-final-final/camunda-modeler-linting-final-final-win-ia32.zip
* https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/linting-final-final/camunda-modeler-linting-final-final-win-x64.zip

File: client/src/app/tabs/panel/tabs/LintingTab.js [hunk 1]
```diff
@@ -31,6 +31,8 @@ export default function LintingTab(props) {
     onAction('showLintError', issue);
   };
 
+  console.log('sorted', JSON.stringify(sortIssues(linting), null, 2));
```
Comment (MaxTru) [inline client/src/app/tabs/panel/tabs/LintingTab.js:34]: Delete me please

**Review by MaxTru — CHANGES_REQUESTED** [review]
Works!
works

Just one minor thing

File: client/src/app/tabs/panel/tabs/LintingTab.js (hunk 1 above)
Comment (philippfromme) [reply client/src/app/tabs/panel/tabs/LintingTab.js:34]: Ahhh! 👿

Comment (philippfromme) [reply client/src/app/tabs/panel/tabs/LintingTab.js]: @MaxTru Done. 👍🏻

**Review by MaxTru — APPROVED** [review]

**Pull request merged**